jsk_perception/CMakeLists.txt

```diff
@@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
+# FIXME: actually, roseus executable is usable only on Linux
+if(GNU)
```

Comment:
> If gdb works, I think we can find the location, if not, we'll check them by running nm -A, and we need to know which libraries are used in roseus, so I'd like to check by
>
ldd roseus/euslisp/roseus.so
``
but could not find ldd 
% brew install ldd

Error: No available formula with the name "ldd" 
==> Searching for similarly named formulae...
This similarly named formula was found:
dcfldd ✔
To install it, run:
brew install dcfldd ✔
==> Searching taps...
Error: No formulae found in taps.

As alternative for `ldd`, you can use `otool -L`.
http://qiita.com/Lewuathe/items/c31edcc9303708cefe3e